scipy/special/_add_newdocs.py

```diff
@@ -12988,8 +12988,41 @@ def add_newdoc(name, doc):
 
     Notes
     -----
-
+    
+    The log function used in the computation is the natural log.
+    
     .. versionadded:: 0.13.0
+    
+    Examples
+    --------
+    
+    Lets see how we can use the function to calculate the following sum: 
+    
+    .. math:: Z = \\sum_{j=0}^n x_j * log(y_j)
```

Comment:
Better, but why someone would want to do this? (A one line of why would be enough.)